runtime/doc/various.txt

```diff
@@ -543,9 +543,19 @@ N  *+X11*		Unix only: can restore window title |X11|
 			bar character.
 
 			The pattern is matched against the relevant part of
-			the output, not necessarily the whole line. Only some
-			commands support filtering, try it out to check if it
-			works.
+			the output, not necessarily the whole line. The
+			following commands support filtering:
```

Comment:
I think `:map`, `:number` (same as `:#`), `:print` and `:z` also support filtering.

Reply:
Thanks I'll update the docs.